.travis.yml

```diff
@@ -13,18 +9,51 @@ cache:
 env:
   global:
     - JACKRABBIT_VERSION=2.12.0
+    - SYMFONY__PHPCR__TRANSPORT=doctrinedbal
+    - TEST_FLAGS=""
+    - SYMFONY__DATABASE__DRIVER=pdo_mysql
+
+matrix:
+  include:
+    - php: 5.5
+    - php: 7.0
+      env: 
+        - SYMFONY__PHPCR__TRANSPORT=jackrabbit
+        # restart jackrabbit after each suite see: https://github.com/sulu-io/sulu/issues/2137
+        - TEST_FLAGS="--jackrabbit-restart"
+    - php: 7.0
+      env:
+        - SYMFONY__DATABASE__DRIVER=pdo_sqlite
+        - SYMFONY__DATABASE__PATH=test.sqlite
```

Comment:
No, its an allowed failure, I have listed the issues we are facing here:
https://github.com/sulu/sulu/issues/2048

On Tue, Apr 05, 2016 at 12:50:55AM -0700, Daniel Rotter wrote:
>    In [1].travis.yml:
> 
>  > +    - SYMFONY__PHPCR__TRANSPORT=doctrinedbal
>  > +    - TEST_FLAGS=""
>  > +    - SYMFONY__DATABASE__DRIVER=pdo_mysql
>  > +
>  > +matrix:
>  > +  include:
>  > +    - php: 5.5
>  > +    - php: 7.0
>  > +      env:
>  > +        - SYMFONY__PHPCR__TRANSPORT=jackrabbit
>  > +        # restart jackrabbit after each suite see: https://github.com/sulu-io/sulu/issues/2137
>  > +        - TEST_FLAGS="--jackrabbit-restart"
>  > +    - php: 7.0
>  > +      env:
>  > +        - SYMFONY__DATABASE__DRIVER=pdo_sqlite
>  > +        - SYMFONY__DATABASE__PATH=test.sqlite
> 
>    SQLite is working now?
> 
>    —
>    You are receiving this because you modified the open/close state.
>    Reply to this email directly or [2]view it on GitHub
> 
>    Reverse link: [3]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/sulu/sulu/pull/2210#discussion_r58499290
>    2. https://github.com/sulu/sulu/pull/2210/files/52e9c12cfcdca3f22eb0ebe24dea839150a2ac60#r58499290
>    3. https://github.com/sulu/sulu/pull/2210/files/52e9c12cfcdca3f22eb0ebe24dea839150a2ac60#r58499290